### PR TITLE
fix: update spdi regex + update tests

### LINKED ISSUE #111: Unable to translate from SPDI deletions
`tlr.translate_from("NC_000017.11:80107579:T:")` returns the following error:

```
File ~/Documents/ga4gh/vrs-python/src/ga4gh/vrs/extras/translator.py:81, in Translator.translate_from(self, var, fmt)
     78         return o
     80 formats = list(self.from_translators.keys())
---> 81 raise ValueError(f"Unable to parse data as {', '.join(formats)}")

ValueError: Unable to parse data as beacon, gnomad, hgvs, spdi, vrs
```

We should update the `spdi_re` in the `Translator` class

### PR DESCRIPTION
**Pull request by korikuzma**
Closes #111 

Added SPDI tests for insertion, deletion, + duplication

### CODE REVIEW

File: tests/extras/test_translator.py [hunk 1]
```diff
@@ -2,188 +2,265 @@
...
+    "state": {
+        "sequence": "C",
+        "type": "LiteralSequenceExpression"
     },
-    'type': 'Allele'
+    "type": "Allele"
+}
+
+# https://www.ncbi.nlm.nih.gov/clinvar/variation/1373966/?new_evidence=true
+deletion_intputs = {
```
Comment (ahwagner) [inline tests/extras/test_translator.py:30]: This looks like a typo, or a clever portmanteau for `int` based inputs.

File: tests/extras/test_translator.py [hunk 2]
```diff
@@ -2,188 +2,265 @@
...
 def test_from_gnomad(tlr):
-    assert tlr._from_gnomad(inputs["gnomad"]).as_dict() == output
+    assert tlr._from_gnomad(snv_inputs["gnomad"]).as_dict() == snv_output
 
 
 @pytest.mark.vcr
 def test_from_hgvs(tlr):
-    assert tlr._from_hgvs(inputs["hgvs"]).as_dict() == output
+    assert tlr._from_hgvs(snv_inputs["hgvs"]).as_dict() == snv_output
+    assert tlr._from_hgvs(deletion_intputs["hgvs"]).as_dict() == deletion_output
```
Comment (ahwagner) [inline tests/extras/test_translator.py:112]: Bonus points for consistency! Beginning to suspect this was intentional...

**Review by ahwagner — APPROVED** [review]
Looks good. Spotted one typo (maybe?) but it is used consistently and it gave me a chuckle so okay to leave or fix at your discretion @korikuzma.

**Comment by korikuzma** [comment]
@ahwagner haha it was a typo. Just fixed in latest commit.

**Pull request merged**